src/main/java/com/starkbank/CorporateCard.java

```diff
@@ -0,0 +1,828 @@
+     * holderId [string]: card holder unique id. ex: "5656565656565656"
+     * <p>
+     * Attributes (return-only):
+     * holderName [string]: card holder name. ex: "Tony Stark"
+     * displayName [string]: card displayed name. ex: "ANTHONY STARK"
+     * rules [list of CorporateRule or HashMap]: [EXPANDABLE] list of card spending rules.
+     * tags [list of strings]: list of strings for tagging. ex: ["travel", "food"]
+     * streetLine1 [string, default sub-issuer street line 1]: card holder main address. ex: "Av. Paulista, 200"
+     * streetLine2 [string, default sub-issuer street line 2]: card holder address complement. ex: "Apto. 123"
+     * district [string, default sub-issuer district]: card holder address district / neighbourhood. ex: "Bela Vista"
+     * city [string, default sub-issuer city]: card holder address city. ex: "Rio de Janeiro"
+     * stateCode [string, default sub-issuer state code]: card holder address state. ex: "GO"
+     * zipCode [string, default sub-issuer zip code]: card holder address zip code. ex: "01311-200"
+     * id [string]: unique id returned when CorporateCard is created. ex: "5656565656565656"
+     * type [string]: card type. ex: "virtual"
+     * status [string]: current CorporateCard status. ex: "active", "blocked", "canceled", "expired"
+     * number [string]: [EXPANDABLE] masked card number. Expand to unmask the value. ex: "123".
+     * securityCode [string]: [EXPANDABLE] masked card verification value (cvv). Expand to unmask the value. ex: "123".
+     * expiration [string]: [EXPANDABLE] masked card expiration datetime. Expand to unmask the value. ex: "2020-03-10 10:30:00.000000+00:00".
+     * updated [string]: latest update datetime for the CorporateCard. ex: "2020-03-10 10:30:00.000000+00:00"
+     * created [string]: creation datetime for the CorporateCard. ex: "2020-03-10 10:30:00.000000+00:00"
+     * @throws Exception error in the request
+     */
+    public CorporateCard(Map<String, Object> data) throws Exception {
+        super(null);
+        HashMap<String, Object> dataCopy = new HashMap<>(data);
+
+        this.holderId = (String) dataCopy.remove("holderId");
+        this.tags = null;
+        this.holderName = null;
+        this.displayName = null;
+        this.rules = null;
+        this.streetLine1 = null;
+        this.streetLine2 = null;
+        this.district = null;
+        this.city = null;
+        this.stateCode = null;
+        this.zipCode = null;
+        this.type = null;
+        this.status = null;
+        this.number = null;
+        this.securityCode = null;
+        this.expiration = null;
+        this.updated = null;
+        this.created = null;
+
+        if (!dataCopy.isEmpty()) {
+            throw new Exception("Unknown parameters used in constructor: [" + String.join(", ", dataCopy.keySet()) + "]");
+        }
+    }
+
+    /**
+     * Create CorporateCard
+     * <p>
+     * Send a CorporateCard object for creation in the Stark Bank API
```

Comment:
Faltou

`If the CorporateCard was not used in the last purchase, this resource will return it.`